### PR TITLE
Fix: Continue cleanup of deployment README.

### PR DESCRIPTION
**Pull request by michaelwsherman**

### CODE REVIEW

File: infra/README.md [hunk 1]
```diff
@@ -125,24 +127,23 @@ Next, we'll incorporate Google Drive details into this file. **Use your preferre
 
 
 ## Application Deployment
-To deploy the backend of the application run the following command from the `/infra` folder. You need to use values output by `terraform apply` for this step.
-
-TODO: make this less unpleasant to do with the values output by terraform apply.
+To deploy the backend of the application run the following command from the `/infra` folder. You need to use values output by `terraform apply` (`region` and `cloud_run_backend_sa`, both without quotes and removing `<` and `>`) for this step.
```
Comment (imp14a) [inline infra/README.md:130]: region is not part of terraform outputs yet, but is part of terraform default variables, currently set to us-central1

Comment (michaelwsherman) [reply infra/README.md:130]: "region" comes out when I run terraform apply.

Comment (imp14a) [reply infra/README.md:130]: you are right I was forgetting that, LGTM

Comment (michaelwsherman) [reply infra/README.md:130]: Confirmed in discussion with @imp14a that region is output from terraform apply.

**Pull request merged**